packages/office-ui-fabric-react/src/components/Slider/Slider.base.tsx

```diff
@@ -46,27 +38,30 @@ export class SliderBase extends BaseComponent<ISliderProps, ISliderState> implem
 
     this.state = {
       value: value,
-      renderedValue: value
+      renderedValue: undefined
```

Comment:
Could/should we also do some sort of warning about the controlled change, or would the list of all those changes belong in a section in the release notes?

Reply:
I'd like to push the warning to another work item and ideally fold in some of the work you have in another branch rather than inflate this PR.